sourcecode/proxies/admin/src/components/Page/Page.jsx

```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles((theme) => ({
         flexGrow: 1,
         padding: theme.spacing(1),
         backgroundColor: '#F6F6F7',
-        minHeight: '100vh',
+        height: 'calc(100vh - 80px)',
```

Comment:
Where does 80px come from? Could this be changed to use a variable?

Reply:
Top menu and padding, updated with an improved solution